src/app/core/services/script-loader/script-loader.service.ts

```diff
@@ -12,16 +14,21 @@ export class ScriptLoaderService {
   // registered scripts
   private scripts: ScriptType[] = [
     {
-      name: 'concardis-payengine',
+      name: 'concardis-payengine-sandbox',
       src: 'https://pptest.payengine.de/bridge/1.0/payengine.min.js',
       loaded: false,
     },
+    {
+      name: 'concardis-payengine-production',
+      src: 'https://pp.payengine.de/bridge/1.0/payengine.min.js',
+      loaded: false,
+    },
   ];
 
   /**
    * load a script, if it has not already been loaded
    */
-  load(name: string): Observable<ScriptType> {
+  load(name: ScriptName): Observable<ScriptType> {
```

Comment:
this makes the `ScriptLoaderService` really special and makes further abstraction harder. I'd favor that the payment-concardis-creditcard.component knows which script to load and the service is more abstract